Assets/Mirror/Components/NetworkRoomPlayer.cs

```diff
@@ -109,12 +109,7 @@ public void CmdChangeReadyState(bool readyState)
         /// <para>This function is called when the a client player calls CmdChangeReadyState.</para>
         /// </summary>
         /// <param name="newReadyState">New Ready State</param>
-        public virtual void ReadyStateChanged(bool _, bool newReadyState)
-        {
-#pragma warning disable CS0618 // Type or member is obsolete
-            OnClientReady(newReadyState);
-#pragma warning restore CS0618 // Type or member is obsolete
-        }
+        public virtual void ReadyStateChanged(bool _, bool newReadyState) { }
```

Comment:
```suggestion
        public virtual void ReadyStateChanged(bool oldReadyState, bool newReadyState) { }
```

adding a name for when this is overriden